Source/FakeItEasy.Specs/ArgumentValueFormatterSpecs.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System.Diagnostics.CodeAnalysis;
+    using FluentAssertions;
+    using Xbehave;
+
+    public static class ArgumentValueFormatterSpecs
+    {
+        [Scenario]
+        public static void GenericArgumentValueFormatterDefaultPriority(
+            IArgumentValueFormatter formatter,
+            Priority priority)
+        {
+            "Given an argument value formatter that extends the generic base"
+                .x(() => formatter = new SomeArgumentValueFormatter());
+
+            "When I fetch the Priority"
+                .x(() => priority = formatter.Priority);
+
+            "Then it should be 0"

```

Comment:
Generally I tend to steer away from stating things which are *not* being done, but in this case I'd be happy with "Given an argument value formatter that extends the generic base without overriding the priority".

Incidentally the initial wording is probably overkill. The base class is already the SUT for the test class, and it is abstract so inheritance from it implies "is a" (as opposed to interface inheritance "can do"), so I think it would be enough to say "Given an argument value formatter that does not override priority".